components/Layout/Footer.tsx

```diff
@@ -1,101 +1,128 @@
             </a>
           </li>
-          <li className="text-white w-64 md:w-56 lg:w-80 my-2.5 hover:underline">
+          <li className="text-white w-64 md:w-56 lg:w-80 hover:underline">
             <a className="font-body" href="#">
               Departments and agencies
             </a>
           </li>
-          <li className="text-white w-64 md:w-56 lg:w-80 my-2.5 hover:underline">
+          <li className="text-white w-64 md:w-56 lg:w-80 hover:underline">
             <a className="font-body" href="#">
               Treaties, laws and regulations
             </a>
           </li>
-          <li className="text-white w-64 md:w-56 lg:w-80 my-2.5 hover:underline">
+          <li className="text-white w-64 md:w-56 lg:w-80 hover:underline">
             <a className="font-body" href="#">
-              How government works
+              About government
             </a>
           </li>
-          <li className="text-white w-64 md:w-56 lg:w-80 my-2.5 hover:underline">
+          <li className="text-white w-64 md:w-56 lg:w-80 hover:underline">
             <a className="font-body" href="#">
               Public service and military
             </a>
           </li>
-          <li className="text-white w-64 md:w-56 lg:w-80 my-2.5 hover:underline">
+          <li className="text-white w-64 md:w-56 lg:w-80 hover:underline">
             <a className="font-body" href="#">
               Government-wide reporting
             </a>
           </li>
-          <li className="text-white w-64 md:w-56 lg:w-80 my-2.5 hover:underline">
+          <li className="text-white w-64 md:w-56 lg:w-80 hover:underline">
             <a className="font-body" href="#">
               Open government
             </a>
           </li>
         </ul>
       </div>
     </div>
-    <div className="my-8">
-      <div className="sm:container mx-auto flex justify-between items-center">
-        <nav className="">
-          <ul className="flex flex-col md:flex-row items-start md:items-center list-disc space-x-6">
-            <li className="list-none">
-              <a href="#">Social media</a>
-            </li>
-
-            <li>
-              <a href="#">Mobile applications</a>
-            </li>
-
-            <li>
-              <a href="#">About Canada.ca</a>
-            </li>
-
-            <li>
-              <a href="#">Terms and conditions</a>
-            </li>
-
-            <li>
-              <a href="#">Privacy</a>
-            </li>
-          </ul>
-        </nav>
-        <div>
-          <Image
-            width="200px"
-            height="40px"
-            src="/wmms-blk.svg"
-            alt="Symbol of the Government of Canada"
-          />
-        </div>
+    <div className="flex justify-between px-4 py-8 sm:container mx-auto">
+      <ul className="flex flex-col md:flex-row md:space-x-6 list-disc px-4 text-[#284162]">
+        <li>
+          <a href="#">Social media</a>
+        </li>
+        #284162
```

Comment:
![image](https://user-images.githubusercontent.com/3630698/151899947-295afc81-ec01-400a-a481-665afbff108c.png)
This doesn't quite seem right! 😉

Reply:
It's better, but you should still remove the bullet point before Social Media, see this screenshot from canada.ca:
![image](https://user-images.githubusercontent.com/3630698/152038925-37aff2bb-d155-48f0-b48d-49b30d7af32e.png)

Reply:
Also, not that it's a problem, but canada.ca has an off-white color, unlike ours which is just plain white.